packages/ava/__tests__/unit/data/statistics/index.test.ts

```diff
@@ -20,7 +24,54 @@ import {
   pearson,
   coefficientOfVariance,
 } from '../../../../src/data/statistics';
-import { cdf, maxabs, normalDistributionQuantile } from '../../../../src/data/statistics/stdlib';
+import {
+  cdf,
+  maxabs,
+  normalDistributionQuantile,
+  vectorAdd,
+  vectorInnerProduct,
+  vectorSubtract,
+  matrixMultiply,
+  multiMatrixMultiply,
+  matrixTranspose,
+  constructDiagonalMatrix,
+  inverseSecondOrderMatrix,
+  tricubeWeightFunction,
+  bisquareWeightFunction,
+  weightedLinearRegression,
+  lowess,
+  pcorrtest,
+} from '../../../../src/data/statistics/stdlib';
+
+expect.extend({
+  toBeCloseToArray(received: number[] | number[][], argument: number[] | number[][]) {
+    const pass = received
+      .map((value: number | number[], i) => {
+        const compareValue = argument[i];
+        if (isArray(value) && isArray(compareValue)) return Math.abs(value[0] - compareValue[0]) < 1e-3;
```

Comment:
`isArray` and `isNumber` are also data utils. Are u sure to use lodash

Reply:
already replaced